integration_tests/suite/helpers/fixtures.py

```diff
@@ -32,6 +36,9 @@ def wrapper(self, *args, **kwargs):
             try:
                 return decorated(self, *output_args, **kwargs)
             finally:
+                if not auto_delete:
+                    return
```

Comment:
This `return` statement make the test always pass ... (thanks to sonarcloud)
I don't understand why we need it? If subscription is deleted by something else, then the 404 is already handled :thinking: 
After removing code, I have this error:
```
E   AttributeError: 'AuthClient' object has no attribute 'set_users'
```
I guess there is a missing PR?

Reply:
Yes, sorry about that. The missing PR: https://github.com/wazo-platform/wazo-test-helpers/pull/118

This is needed because we're removing the tenant in a test, so the test code will receive a 401 error when deleting the subscription of the deleted tenant.